app/scripts/services/room.js

```diff
@@ -3,9 +3,9 @@
     var ref = firebase.database().ref().child("rooms");
     var rooms = $firebaseArray(ref);
       
-    this.addRoom = function(newRoom) {
-        rooms.$add(newRoom)
-    }  
+    this.addChatRoom = function(newChatRoomName) {
+        rooms.$add({$value: newChatRoomName});
```

Comment:
So, the name for a new chat will be passed to the CreateRoomCtrl in the form of newChatRoomName which will then be passed to the addChatRoom method which will then add that name to the firebase array?